src/dune_pkg/rev_store.ml

```diff
@@ -1156,7 +1163,7 @@ let resolve_revision t (remote : Remote.t) ~revision =
 let fetch_object t (remote : Remote.t) revision =
   fetch_allow_failure t ~url:remote.url revision
   >>= function
-  | `Not_found -> Fiber.return None
+  | `Not_found _ -> Fiber.return None
```

Comment:
I don't think you want to throw away the error here.

Reply:
Yes, makes sense. I pushed a commit to handle the error in fetch_object too. Thanks!